app/routes/about/About.jsx

```diff
@@ -7,11 +11,42 @@ export function meta() {
   ];
 }
 
-export default function About() {
+export default function about() {
   return (
     <>
-      <h1>About</h1>
-      <Outlet />    
+      <div className="bg-white text-center justify-center items-center">
+        <div
+          className="text-center justify-center w-full bg-no-repeat bg-cover bg-center"
+          style={{ backgroundImage: `url(${AboutBanner})` }}
+        >
+          <h1 className="font-[Julius_Sans_One] text-black text-6xl uppercase bg-white/40 w-full h-full p-32">
```

Comment:
<img width="314" alt="image" src="https://github.com/user-attachments/assets/8beffdd6-633c-4620-aed6-23dec0da5dcf" />


We can make the text smaller for mobile para it fits :D